revenuecatui/src/androidMain/kotlin/com/revenuecat/purchases/kmp/ui/revenuecatui/OriginalTemplatePaywallFooter.kt

```diff
@@ -2,15 +2,13 @@ package com.revenuecat.purchases.kmp.ui.revenuecatui
 
 import androidx.compose.foundation.layout.PaddingValues
 import androidx.compose.runtime.Composable
-import com.revenuecat.purchases.ui.revenuecatui.ExperimentalPreviewRevenueCatUIPurchasesAPI
 import com.revenuecat.purchases.ui.revenuecatui.PaywallFooter as AndroidPaywallFooter
 
 /**
  * A Paywall footer.
  */
-@OptIn(ExperimentalPreviewRevenueCatUIPurchasesAPI::class)
 @Composable
-public actual fun PaywallFooter(
+public actual fun OriginalTemplatePaywallFooter(
```

Comment:
Note that we should update the references to the deprecated APIs in the natives once the natives have shipped and have been propagated (same applies to the other hybrids), but this PR contains the changes to the public API, the rest can be done at any point in time